### DISCUSSION TITLE
Auto saving last mods

### DISCUSSION THREAD

**Discussion by deissh** [ppy/osu#13062] [Ideas] [answered]
Probably everyone who plays with mods enters F1 and enables the necessary mods every time.
I suggest saving the last selected mods and by default turning them on when we enter the game.

**Comment by frenzibyte** [comment]
What about making something like mod combinations where you can save your own list of mods and then when selecting the combination, it loads these mods ;)

**Comment by EVAST9919** [comment]
Probably everyone is not everyone. At least should be an option in settings IMO

**Comment by peppy** [comment, marked as answer]
I believe this is being considered in @arflyte's redesign of the mod select overlay.

**Comment by AuracleTech** [comment]
I play with diff adjust and approach different
Setting em up everyday is pain

Could be epic if the game had a new setting 

- [ ] Restore previous mods at launch

**Reply by integerrr** [reply]
If you don't have anything tangible to add other than what's been mentioned then please refrain from posting these "+1" comments. Like what's mentioned above, the new mod select screen will address this issue with mod presets/loadouts

**Reply by AuracleTech** [reply]
> please refrain from posting these "+1" comments

I'm sorry, are we on stackoverflow? Or did I miss click the discussion tab and clicked on issues?

I could not re-open a thread with the same purpose. Also this one is marked as `solved` and buried deep.

My only recourse is to try and promote this issue which is also my issue.

The difference between a +1 and a comment is that even if a post had 1 million likes, the thread could be buried deep.
The bump is the difference.

Don't hate the player hate the game. I'm not the one that made bumps non available without writing a message.
I recommend you visit github if you have issues with the way users interact with the website.

**Reply by EVAST9919** [reply]
"Bumps" are doing nothing more here than distracting devs from fixing other issues. Answer has been already given and even if it wasn't - that means that the issue isn't worth looking into rn. So please just be patient.

**Reply by frenzibyte** [reply]
@AuracleTech Please cut the attitude. As has already been mentioned, what you're suggesting is a simplified version of mod presets, and mod presets will eventually come seeing that the redesigned mod select overlay is making its way through. 

There's *no* point in "trying and promoting" this issue while the redesigned overlay hasn't been integrated to the game yet, as that'll just be dismissed. A better approach would've been opening a discussion thread tracking the new feature after the overlay has been integrated to the game, to give a reason for tracking it.

**Reply by AuracleTech** [reply]
Its not even promoting, this issue is from 3 years ago so it was buried, I just wanted to bump it.
But I get the point you're making.

**Reply by peppy** [reply]
Bumping discussions and issues are fine on very old discussions, especially if new information or use cases are added.

That said, in a case like this we do have ongoing work on a new mod select (as mentioned in the most recent post in this very discussion thread) which already explains that this is being considered.

Would be appreciated if you can read into that a bit.
